.travis.yml

```diff
@@ -63,6 +63,12 @@ jobs:
     - name: "unit test, integration test edge"
       arch: amd64
       before_script:
+        - sudo apt-get update
+        - sudo apt-get install apt-transport-https ca-certificates curl gnupg-agent software-properties-common -y
+        - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+        - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+        - sudo apt-get update
+        - sudo apt-get install docker-ce docker-ce-cli containerd.io -y
```

Comment:
For higher version of docker? Do you try with traivs config instead of installing directly?

Reply:
The travisCI docker version comes with travisCI Ubuntu 18.04 vm. If we don't update VM OS, TravisCI recommends to update ourselves. So we did it in the test setup.

Reply:
Just update travis config, see [doc](https://docs.travis-ci.com/user/reference/linux/#using-xenial), no need to update during test.

Reply:
Which version of docker do we need now? Maybe need comment it in doc.

Reply:
@daixiang0 , which Doc do you suggest to add the version to? Readme.md? or some other place (but it may easily be overlooked)?

Currently: 
docker-ce  5:19.03.13
docker-ce-cli  5:19.03.13
containerd.io  1.3.7-1

Reply:
In docs/configuration/cri.md, add docker section.

Reply:
I think set focal in travis config is better, after upgrade to 1.19, all test should run with docker 19.03+.

Has checked it, the default docker version of focal is 19.03.

Reply:
Updated TravisCI base VM to focal, and it passed all tests. I will create a separate PR to update the doc to save time on this PR. Thanks.